From 5b69f167ca48e437a3bb6fb7f93d6557259aecad Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 10 Jul 2018 11:58:48 +0300
Subject: [PATCH] dali-<board>-tbot: add support for PCIE_HOST_TBOT_IP

The ip / hostname is used by tbot to connect for firmware preparation
and upload to the dali board via the PCIe.
Add default setting and an ability to override from the environment.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 5cefee7..e9976d8 100644
--- a/Makefile
+++ b/Makefile
@@ -31,6 +31,8 @@
 	@echo '                             override dafault test tc_ub_help.py'
 	@echo '                           set HOST_TBOT_IP=<PC lab IP> to'
 	@echo '                             override <board> dafault IP'
+	@echo '                           set PCIE_HOST_TBOT_IP=<PCIe host IP> to'
+	@echo '                             override <board> dafault IP'
 
 export BR2_EXTERNAL := $(CURDIR)
 OUTPUT_DIR := $(BR2_EXTERNAL)/output
@@ -146,16 +148,20 @@
 .PHONY: dali-onix-tbot dali-pldm-tbot dali-asic-tbot
 dali-onix-tbot: dali-onix-tools-install
 dali-onix-tbot: HOST_TBOT_IP ?= monster4-nuc
+dali-onix-tbot: PCIE_HOST_TBOT_IP ?= monster4
 dali-pldm-tbot: dali-pldm-tools-install
 dali-pldm-tbot: HOST_TBOT_IP ?= 192.168.10.45
+dali-pldm-tbot: PCIE_HOST_TBOT_IP ?= nohost
 dali-asic-tbot: dali-asic-tools-install
 dali-asic-tbot: HOST_TBOT_IP ?= 192.168.10.109
+dali-asic-tbot: PCIE_HOST_TBOT_IP ?= monster5
 dali-onix-tbot dali-pldm-tbot dali-asic-tbot:
 	@HOST_TBOT_LAB=lab_dali					\
 	HOST_TBOT_CFG=dali					\
 	HOST_TBOT_BRD=$(patsubst dali-%-tbot,%,$@)		\
 	HOST_TBOT_CASE=$(HOST_TBOT_TC)				\
 	HOST_TBOT_LAB_IP=$(HOST_TBOT_IP)			\
+	PCIE_HOST_TBOT_LAB_IP=$(PCIE_HOST_TBOT_IP)		\
 		$(MAKE) host-tbot-rebuild
 dali-pldm-tools-install: DALI_INSTALL_LAB_IP ?= 192.168.10.45
 .PHONY: dali-pldm-tools-install
